KesaApplication/ViewController.swift

```diff
@@ -10,16 +10,130 @@ import UIKit
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+    // 4 labels
+    @IBOutlet weak var firstNameLabel: UILabel!
+    @IBOutlet weak var lastNameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet weak var passwordLabel: UILabel!
+    
+    // 4 text fields
+    @IBOutlet weak var firstName: UITextField!
+    @IBOutlet weak var lastName: UITextField!
+    @IBOutlet weak var email: UITextField!
+    @IBOutlet weak var password: UITextField!
+    
+    
+    // When the login button is pressed, ignore all text fields
+    @IBAction func loginButtonPressed(sender: UIButton) {
+        self.firstName.resignFirstResponder() // don't worry about the keyboard
+        self.lastName.resignFirstResponder()
+        self.email.resignFirstResponder()
+        self.password.resignFirstResponder()
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+    
+    // When the screen is pressed while editing, hide the keyboard
+    override func touchesBegan(touches: Set<UITouch>, withEvent event: UIEvent?) {
+        self.view.endEditing(true)
+        firstNameLabel.textColor = UIColor.blackColor()
+        lastNameLabel.textColor = UIColor.blackColor()
+        emailLabel.textColor = UIColor.blackColor()
+        passwordLabel.textColor = UIColor.blackColor()
+    }
+    
+    
+    
+    // Highlighting each labels when their text fields are pressed
+    @IBAction func firstNameTextFieldPressed(sender: UITextField) {
+        firstNameLabel.textColor = UIColor.orangeColor()
+        lastNameLabel.textColor = UIColor.blackColor()
+        emailLabel.textColor = UIColor.blackColor()
+        passwordLabel.textColor = UIColor.blackColor()
+    }
+    
+    @IBAction func lastNameTextFieldPressed(sender: UITextField) {
+        firstNameLabel.textColor = UIColor.blackColor()
+        lastNameLabel.textColor = UIColor.orangeColor()
+        emailLabel.textColor = UIColor.blackColor()
+        passwordLabel.textColor = UIColor.blackColor()
     }
 
-
+    
+    @IBAction func emailTextFieldPressed(sender: UITextField) {
+        firstNameLabel.textColor = UIColor.blackColor()
+        lastNameLabel.textColor = UIColor.blackColor()
+        emailLabel.textColor = UIColor.orangeColor()
+        passwordLabel.textColor = UIColor.blackColor()
+    }
+    
+    @IBAction func passwordTextFieldPressed(sender: UITextField) {
+        firstNameLabel.textColor = UIColor.blackColor()
+        lastNameLabel.textColor = UIColor.blackColor()
+        emailLabel.textColor = UIColor.blackColor()
+        passwordLabel.textColor = UIColor.orangeColor()
+    }
+    
+    
+    
+    // When "Create Account" button is pressed
+    @IBAction func createAccountButtonPressed(sender: UIButton) {
+        
+        let userFirstName = firstName.text
```

Comment:
I think it will be better to check one by one.
1. Check whether the first name or last name contains any number or empty
2. Check the email format